src/components/victory-candlestick/victory-candlestick.js

```diff
@@ -455,10 +467,12 @@ export default class VictoryCandlestick extends React.Component {
   }
 
   render() {
+    const modifiedProps = this.props.theme ? Helpers.modifyProps(this.props, fallbackProps,
+      this.props.theme.candlestick.props) : Helpers.modifyProps(this.props, fallbackProps);
```

Comment:
If you're passing `this.props` into `modifyProps`, you shouldn't need this extra arg.